python/prophet/diagnostics.py

```diff
@@ -58,7 +58,7 @@ def generate_cutoffs(df, horizon, initial, period):
     return list(reversed(result))
 
 
-def cross_validation(model, horizon, period=None, initial=None, parallel=None, cutoffs=None, disable_tqdm=False):
+def cross_validation(model, horizon, period=None, initial=None, parallel=None, cutoffs=None, disable_tqdm=False, predict_columns=['ds','yhat']):
```

Comment:
I suggest making it None by default
```suggestion
def cross_validation(model, horizon, period=None, initial=None, parallel=None, cutoffs=None, disable_tqdm=False, predict_columns=None):
```

and dealing with it after, e.g.

```
if predict_columns is None:
    predict_columns = ['ds', 'yhat']
```
as in general the default params aren't to be mutable. 

The introduced parameter also needs to be documented.

Reply:
Thanks for the suggestion! Updated the PR